packages/terra-switch/src/Switch.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useRef, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { FormattedMessage } from 'react-intl';
+import { enableFocusStyles, disableFocusStyles, restoreFocusStyles } from './_SwitchUtils';
+import styles from './Switch.module.scss';
+
+const cx = classNames.bind(styles);
+
+const SWITCH_STATE = Object.freeze({
+  ON: <FormattedMessage id="Terra.switch.switchstatuslabel.on" />,
+  OFF: <FormattedMessage id="Terra.switch.switchstatuslabel.off" />,
+});
+
+const propTypes = {
+  /**
+   * Id of the Switch button.
+   */
+  switchId: PropTypes.string.isRequired,
+  /**
+   * Whether or not the Switch is checked ("ON").
+   */
+  checked: PropTypes.bool,
```

Comment:
```suggestion
  isOn: PropTypes.bool,
```

Reply:
It was isOn initially but later I changed it checked w.r.t this [comment](https://github.com/cerner/terra-core/pull/2846#discussion_r383431728)

Reply:
To align with our standards, we should be used prepending with `is` or `has` for boolean props since this isn't a form component html attribute: https://engineering.cerner.com/terra-ui/about/terra-ui/contributing/conventions#for-boolean-props-define-defaultprop-value-as-false

Can we update to `isChecked`?

Reply:
updated [here](https://github.com/cerner/terra-core/pull/2846/commits/8ba4fb42d6a30245f9e12acc308252d47b22d7d2#diff-ce03daf8e4362de3a438b2f322dedf6f)